### ISSUE TITLE
How to map local workspace subfolder to remote path?

### ISSUE THREAD

**Issue by SynCap** [lukasz-wronski/vscode-ftp-sync#140]
Only one of subfolders is have to be uploaded/synced to/with remote server.
How to do this?
How about to map several subfolders to different remote place? Also that may be a different connections (servers) of course, but may be same server.

**Comment by nikolaydyankov** [comment]
Yes, I don't know why all extensions are missing this feature, it's really useful to have. I've seen it only in Sublime.

**Comment by rastographics** [comment]
Someone already [did a pull request](https://github.com/lukasz-wronski/vscode-ftp-sync/pull/51) for this but it has not been merged yet due to conflicts...

I'm eagerly awaiting this too....

**Comment by ChaimHong** [comment]
Yes, I don't know why all extensions are missing this feature, it's really useful to have.
